mypy/checkexpr.py

```diff
@@ -3294,19 +3313,25 @@ def visit_dict_expr(self, e: DictExpr) -> Type:
         assert rv is not None
         return rv
 
-    def find_typeddict_context(self, context: Optional[Type]) -> Optional[TypedDictType]:
+    def find_typeddict_context(self, context: Optional[Type],
+                              dict_expr: DictExpr) -> Optional[TypedDictType]:
```

Comment:
(Has the linter broken??). Indent `dict_expr` another space

Reply:
Yeah this was weird, my vscode showed this being inline! I've added the space in but yeah weird this wasn't picked up.